apis/swagger.yml

```diff
@@ -963,9 +970,9 @@ definitions:
       callSystem:
         type: "string"
         description: |
-          This field is for debugging. When caller of dfget is using it to files, he can pass callSystem
-          name to dfget. When this field is passing to supernode, supernode has ability to filter them via 
-          some black/white list to guarantee security, or some other purposes.
+          This attribute represents where the dfget requests come from. Dfget will pass
```

Comment:
@lowzj  I don't think the `callSystem` and `dfdaemon` should be held by taskInfo. And maybe more reasonable for `dfgetTask`. WDYT?

Reply:
I think that it's necessary to pass the `callsystem` parameter to supernode when register a task, but there's no need to store it in task info.

Reply:
@yeya24 Could you please make a change for that?
+ delete the `dfdaemon` and `callSystem` fields from taskInfo
+ add them to `dfgetTask`

Reply:
@Starnop  Done. PTAL.